live-examples/js-examples/array/.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+// We use `.prettierrc.js` instead of `.prettierrc.json`
+// so we can extend this repository's default `prettier` configuration.
+module.exports = {
+  ...require('../../../.prettierrc.json'),
+  printWidth: 100,
```

Comment:
```suggestion
  // This prevented at least one example from requiring horizontal scrolling.
  printWidth: 100,
```

Reply:
It might be worth documenting why we're overriding the base `prettier` config.

Reply:
So it appears I set the [print width to 120](https://github.com/mdn/interactive-examples/pull/2274#issuecomment-1268508109) which might not even be desirable overall. It looks like the original objection to `80` as a print width was for changes in an `.md` file (which we can maybe handle with overrides?), not a code example. It might be worth considering getting rid of the `printWidth` key altogether and using the default which is [what we use in content](https://github.com/mdn/content/blob/main/.prettierrc.json) with overrides for html/md. We should check how many files this changes though